### ISSUE TITLE
Obsoleted shader functions "textureScreen" and "SRC_COLOR" still show in autocomplete

### ISSUE THREAD

**Issue by scayze** [godotengine/godot#11383]
**Operating system or device, Godot version, GPU Model and driver:**
Ubuntu 16.04
Godot 3.0 bc005a57d38137b062e0ccc26467353f53556930
GPU: AMD RX 480 with AMDGPU driver

**Issue description:**
The canvas_item shader:
```
shader_type canvas_item;

void fragment() {
	COLOR = textureScreen(UV);
}
```
on a sprite node crashed with:

`ERROR: _display_error_with_code: CanvasShaderGLES3: Fragment Program Compilation Failed:
0:238(8): error: no function with name 'textureScreen'`

**Steps to reproduce:**
- Create a sprite node
- attatch a Shader material
- Write a shader that utilizes the `textureScreen` method in any way
- Profit?

**Comment by reduz** [comment]
this function no longer exists in 3.0. use:

texture(SCREEN_TEXTURE,SCREEN_UV)

**Comment by scayze** [comment]
ok, thank you! The function still shows up on autocomplete though. same with SRC_COLOR (#11233), which, according to the new shader docs, doesnt exist anymore either
@reduz

**Comment by akien-mga** [comment]
Repurposing this issue to be about the autocomplete problem mentioned above, which seems not to match the functions of 3.0.

**Comment by akien-mga** [comment]
We have now entered release freeze for Godot 3.0 and want to focus only on release critical issues for that milestone. Therefore, we're moving this issue to the 3.1 milestone, though a fix may be made available for a 3.0.x maintenance release after it has been tested in the master branch during 3.1 development. If you consider that this issue is critical enough to warrant blocking the 3.0 release until fixed, please comment so that we can assess it more in-depth.

**Comment by scayze** [comment]
Not reproducible anymore.
